docs/content/standards/currency.mdx

```diff
@@ -35,56 +50,56 @@ The registry is a system-level shared object located at address `0xc`.
 <ImportContent source="crates/sui-framework/packages/sui-framework/sources/registries/coin_registry.move" mode="code" />
 </details>
 
-#### Core components
+### Core components
 
-**`CoinRegistry`:** The main system object that coordinates all currency-related operations. This shared object serves as the entry point for all registry operations and is created once during network initialization. It's address is `0xc`.
+**`CoinRegistry`:** The main system object that coordinates all coin-related operations. This shared object serves as the entry point for all registry operations and is created once during network initialization. Its address is `0xc`.
 
 **`Currency<T>`:** The core of the registry system, storing comprehensive information about each coin type including:
-- **Metadata management:** Basic currency information like name, symbol, decimals, description, and icon URL.
+- **Metadata management:** Basic coin information like name, symbol, decimals, description, and icon URL.
 - **Supply tracking:** Maintains supply state information (fixed, burn-only, or unknown).
 - **Regulatory status:** Tracks whether the coin is regulated with deny list capabilities.
-- **Capability references:** Links to `TreasuryCap`s and `MetadataCap`s for the coin type.
+- **Capability references:** Links to treasury and metadata capabilities for the coin type.
 - **Extensibility:** Includes extra fields for future enhancements.
 
-#### Supply states
+### Supply states
 
 The registry supports three different supply management models:
 
 - **Fixed supply:** The total supply is permanently locked and cannot be changed.
 - **Burn-only supply:** New coins cannot be minted, but existing coins can be burned.
 - **Uncontrolled supply:** `TreasuryCap` holder controls minting and burning.
 
-#### Regulatory states
+### Regulatory states
 
-Currencies can have different regulatory states:
+Coins can have different regulatory states:
 
-- **Regulated:** The currency has an associated `DenyCapV2` that can restrict addresses from using it.
-- **Unregulated:** The currency was created without any deny list capabilities.
+- **Regulated:** The coin has an associated `DenyCapV2` that can restrict addresses from using it.
+- **Unregulated:** The coin was created without any deny list capabilities.
 - **Unknown:** Regulatory status is undetermined, typically from legacy migrations.
 
-#### Creation options
+### Creation options
 
-The registry supports two different currency creation flows:
+The registry supports two different coin creation flows:
```

Comment:
Again we're at Coin <-> Currency debate. I'm good with either, but worth having a broader discussion what we call what. Because:

- Coin is a single object wrapper for a fungible asset
- Currency is the object that is created in CoinRegistry which describes the currency properties and setup
- Both coin type and currency type are legit, with the former being dominant in the space